.github/workflows/license-reusable.yml

```diff
@@ -70,32 +59,24 @@ jobs:
         pip3 install --user -U west
         pip3 show -f west
 
-    - name: Git config and rebase
+    - name: Git rebase
       env:
         BASE_REF: ${{ github.base_ref }}
       working-directory: ncs/${{ inputs.path }}
       run: |
-        git config --global user.email "you@example.com"
-        git config --global user.name "Your Name"
         git remote -v
         git branch
         git rebase origin/${BASE_REF}
         # debug
         git log --pretty=oneline | head -n 10
 
-    - name: West init and update
+    - name: West zephyr-export
       working-directory: ncs
       env:
         PR_REF: ${{ github.event.pull_request.head.sha }}
       run: |
         export PATH="$HOME/.local/bin:$PATH"
         export PATH="$HOME/bin:$PATH"
-        if [ "${{ inputs.path }}" = "nrf" ]; then
-            west init -l nrf
-        else
-            west init -m ${{ inputs.nrf_repo }} --mr ${{ inputs.nrf_rev }}
```

Comment:
It looks like the new action does not support this use case.

Reply:
Correct. I tried to search if these inputs are actually used somewhere but didn't find hits (but I might have missed something).

Reply:
Checked as well and couldn't find it being used.